rules/do-you-know-how-to-claim-expense-reimbursements-2/rule.md

```diff
@@ -50,54 +40,32 @@ At SSW we find the greatest overall value for expense reimbursements is to use t
  **Employees:** 
 
 
-
-
-  1. Using the Xero Expense mobile app, submit a photo of the receipt. If the receipt was sent via email, you can still submit a photo or screenshot of the email receipt
-
-
-
-
-
-
-
+1. Open the Xero Expense mobile app and submit a photo of the receipt for reimbursement or filing. If the receipt was sent via email, you can still submit a photo or screenshot of the email receipt
```

Comment:
I'm a little confused as to the purpose of the second sentence here. Could you elaborate for me on what it is trying to say?

Reply:
The second sentence should be saying that a digital receipt can still be uploaded manually instead of using the 'take photo' function of the app. 

Perhaps it could be worded as, "For digitally emailed receipts, you can either take a photo of the email and upload it as per usual or you can upload the PDF version of it within the app."

Reply:
I think that wording would be better.